identity-wallet/src/state/search/mod.rs

```diff
@@ -0,0 +1,20 @@
+pub mod actions;
+pub mod reducers;
+
+use super::FeatTrait;
+
+use derivative::Derivative;
+use serde::{Deserialize, Serialize};
+use ts_rs::TS;
+
+#[derive(Default, Serialize, Deserialize, Derivative, TS, Clone, PartialEq, Debug)]
```

Comment:
The `Derivative` derive can be used to add more fine-grained attributes to struct. However, they are not being utilized here so better to remove `Derivative` from the derive list :+1: